### PR TITLE
Change order of getting started walkthrough steps

### PR DESCRIPTION
**Pull request by bhavyaus**

### CODE REVIEW

File: src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts [hunk 1]
```diff
@@ -180,6 +180,16 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 		content: {
 			type: 'steps',
 			steps: [
+				{
+					id: 'settingsSync',
+					title: localize('gettingStarted.settingsSync.title', "Sign in to get started with VS Code"),
+					description: localize('gettingStarted.settingsSync.description.interpolated', "Keep your essential VS Code customizations backed up and updated across all your devices.\n{0}", Button(localize('enableSync', "Enable Settings Sync"), 'command:workbench.userDataSync.actions.turnOn')),
```
Comment (sandy081) [inline src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts:186]: I would rename the button to - **Backup and Sync Settings** to be consistent.

File: src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts [hunk 2]
```diff
@@ -180,6 +180,16 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 		content: {
 			type: 'steps',
 			steps: [
+				{
+					id: 'settingsSync',
+					title: localize('gettingStarted.settingsSync.title', "Sign in to get started with VS Code"),
```
Comment (sandy081) [inline src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts:185]: May be we should say here **Personalise your VS Code** or **Make VS Code your own** instead?

**Review by sandy081 — CHANGES_REQUESTED** [review]

**Comment by sandy081** [comment]
CC @isidorn

**Comment by sandy081** [comment]
It would be great if we can get a fresh picture here

image

File: src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts (hunk 2 above)
Comment (isidorn) [reply src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts:185]: "Personalise your VS Code" sounds good to me.

**Comment by isidorn** [comment]
@bhavyaus looks good! Once we merge this in I can test it out and provide more feedback.
cc @hbons for new image if needed for the welcome flow

**Review by sandy081 — APPROVED** [review]

**Pull request merged**